static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -12,8 +12,19 @@ import type { PrebidSize } from './types';
 const isInAppnexusUSAdapterTestVariant = (): boolean =>
     isInVariantSynchronous(appnexusUSAdapter, 'variant');
 
+const SUFFIX_REGEXPS = {};
 const stripSuffix = (s: string, suffix: string): string => {
-    const re = new RegExp(`${suffix}$`);
+    const re =
+        SUFFIX_REGEXPS[suffix] ||
+        (SUFFIX_REGEXPS[suffix] = new RegExp(`${suffix}$`));
+    return s.replace(re, '');
+};
+
+const PREFIX_REGEXPS = {};
+const stripPrefix = (s: string, prefix: string): string => {
```

Comment:
Do these need tests?

Reply:
They are already covered by the utils tests.